canonical/canonical.com
# WD-4092 Update careers departments

## Done

Update design and refactor for consistency

## QA

- Check all department pages look consistent now:
- Go to https://canonical-com-951.demos.haus/careers and click on each department.
